src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3379,7 +3379,7 @@ char* os::pd_attempt_reserve_memory_at(char* addr, size_t bytes, bool exec) {
     }
     if (Verbose && PrintMiscellaneous) {
       reserveTimer.stop();
-      tty->print_cr("reserve_memory of %Ix bytes took " JLONG_FORMAT " ms (" JLONG_FORMAT " ticks)", bytes,
+      tty->print_cr("reserve_memory of %zx bytes took " JLONG_FORMAT " ms (" JLONG_FORMAT " ticks)", bytes,
```

Comment:
*If* we change anything here, the correct format would be SIZE_FORMAT.

Reply:
Regarding SIZE_FORMAT - https://bugs.openjdk.org/browse/JDK-8256379.  Some people have been replacing
uses of SIZE_FORMAT when touching the relevant code for other reasons.

Reply:
Yes, I think it might be best if people started using %zd rather than adding new SIZE_FORMAT to the code. I haven't figured out the best way to change the rest.  If using %zx, I suggest adding 0x%zx.